### ISSUE TITLE
Input to search through Select options loses focus on every keystroke

### ISSUE THREAD

**Issue by alessandrocapra** [GeekyAnts/NativeBase#5610]
### Description

After opening the Select and startin to type in the Input field to filter the options, the keyboard gets dismissed and the field loses focus

### CodeSandbox/Snack link

https://snack.expo.dev/@alessandro_capra/af3ece

### Steps to reproduce

- Open the provided Snack link
- Open the country Select input
- Try to type in the search field

**Expect**
- iOS: kinda working, but flickering in the background
- Android: not working, the Input loses focus and the keyboard is dismissed

### NativeBase Version

3.4.13

### Platform

- [X] Android
- [ ] CRA
- [X] Expo
- [X] iOS
- [ ] Next

### Additional Information

I believe it's related to the ActionSheet focusing automatically on its first element every time the component rerenders?
I tried using an uncontrolled Input but without any luck, any suggestion?

**Comment by ruchik02** [comment]
Thanks for reporting . We will look into this

**Comment by alessandrocapra** [comment]
Thanks for the reply @ruchik02 :) Do you have an approximate ETA for this bug? I can try to give you more details if needed, I thought it would be quite a pretty common use case but it doesn't seem like it? 

Once it's fixed, I can add some content to the docs to help out ;)

**Comment by ruchik02** [comment]
Hey @alessandrocapra  I see this issue but in select component this feature is not added, may be in future will add this  feature to our list.

**Comment by alessandrocapra** [comment]
Can you explain what you mean by "it's not added"? Does it mean that this bug is not going to be addressed? I just need to know it, in case I need to drop this implementation because nobody is going to look at it..

Is there any workaround that you might be able to provide?

**Comment by ruchik02** [comment]
No, no, I mean in select component like you start in type in Select in component this feature is currently not added 
@alessandrocapra

**Comment by alessandrocapra** [comment]
I'm sorry but I have a hard time understanding your message. What I read is that my use case is currently not an official feature, and you will work on adding it in the future, correct?

So my follow-up question is, would there me a workaround? Or should I not try what I am doing here and just use something else (not NativeBase)?

Thanks again

**Comment by mrkpatchaa** [comment]
Is there a way to disable this behavior? I just want a simple select without the option to search by typing.

**Comment by wenotch** [comment]
please how do I disable the input
